### PR TITLE
feat: Created onboarding screens for app

### PR DESCRIPTION
**Pull request by imjog**
**Fixes** #269 

**Changes:**

Designed and implemented onboarding screens for the app from scratch.
- There are **four** onboardings screens.
- Each screen contains an `image`, `title`, and `description` about SUSI iOS features.
- **Skip** button to directly go to Login screen - on the top for first three and on the bottom for last one.
- Onboarding will only show once when the user first time launches the app on the device. It uses `lanuchedBefore` UserDefaults key to store information about user launches app for first time or not.
- Left and Right **Swiping is also enabled**; user can go forward and backward to onboarding screens by swiping left and right.
- **UI Test** - Added `OnboardingUITests`.
- All the onboarding screens are using **Material** design.

**Screenshots for the change:** 

<table>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
</tr>
</table>

### CODE REVIEW

**Comment by mariobehling** [comment]
Looks good to me from the UI. Please check circle-ci tests.

**Comment by mariobehling** [comment]
Please squash commits.

**Comment by imjog** [comment]
@mariobehling I had that in mind but I planned to squash it after passing the tests.  I think UI tests are failing because I add a new screen before the Login screen, I am looking into that. I will squash commits after that. Thank you

**Comment by chashmeetsingh** [comment]
@imjog please add a WIP label in the title of the label until its complete.

**Comment by imjog** [comment]
@chashmeetsingh I think now the problem is from the `circle ci`, their 2.0 workflow is not working currently.
- I write new UI tests for Onboarding screens
- Modified the other UI test according to onboarding screens
- Circle-CI is showing error:

> /Users/distiller/susi_iOS/Pods/BouncyLayout/BouncyLayout/Classes/BouncyLayout.swift:49:137: value of type '[UICollectionViewLayoutAttributes]' has no member 'compactMap'

This is because **BouncyLayout**  pod is using swift 4.1. `flatMap` is deprecated (swift 4.0) and replaced with `compactMap` in swift 4.1 so I updated the circle ci xcode, OS and swift version but they are still failing due to same error.
Currently our project is using Swift 4.1

**Comment by chashmeetsingh** [comment]
@imjog Please make a dummy commit to restart the build. We need to have the build pass all the tests so that we can merge the PR.

File: circle.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 machine:
```
Comment (chashmeetsingh) [inline circle.yml]: Also, it seems that Circle CI version 1 is being used right now. Add `version: 2` at the top of this file to migrate to v2.0 [source](https://circleci.com/docs/2.0/sample-config/)

**Comment by chashmeetsingh** [comment]
Also, it seems that Circle CI version 1 is being used right now. Add `version: 2` at the top of `config.yml` file to migrate to v2.0 source

**Comment by mariobehling** [comment]
@imjog Thanks. Please work with other devs to get approvals as well. Let's proceed more in a community way. Anyone can make a review.

File: circle.yml (hunk 1 above)
Comment (imjog) [reply circle.yml]: Made the required changes in #289

**Comment by imjog** [comment]
@SIMRAN1 Please review the PR. @chashmeetsingh Please look into "The circleci showing - Expected — Waiting for status to be reported."
Please resolve this by following https://circleci.com/docs/2.0/workflows-waiting-status/. I do not have access.

**Comment by chashmeetsingh** [comment]
Please squash your commits.

**Comment by imjog** [comment]
@chashmeetsingh All the commits except merge commits are squashed.

**Pull request merged**